sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -110,6 +110,26 @@ static TestEnvironment()
         /// </summary>
         public string ClientSecret => GetVariable("CLIENT_SECRET");
 
+        /// <summary>
+        ///   The authority host of the cloud to use during Live tests. Recorded.
+        /// </summary>
+        public string AuthorityHost => GetRecordedVariable("AZURE_AUTHORITY_HOST");
+
+        /// <summary>
+        ///   The service management Url of the cloud to use during Live tests. Recorded.
+        /// </summary>
+        public string ServiceManagementUrl => GetRecordedVariable("SERVICE_MANAGEMENT_URL");
+
+        /// <summary>
+        ///   The resource manager of the cloud to use during Live tests. Recorded.
+        /// </summary>
+        public string ResourceManager => GetRecordedVariable("RESOURCE_MANAGER");
```

Comment:
Given the usage of this, would it be better as a `Uri` here, rather than being parsed each time it is used?